channels/lb.m3u

```diff
@@ -31,6 +31,8 @@ https://svs.itworkscdn.net/nour8satlive/livestream/playlist.m3u8
 https://svs.itworkscdn.net/nour9satlive/livestream/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Arabic" tvg-logo="https://i.imgur.com/QrwZu4W.png" group-title="Religious",NourSAT (576p)
 https://svs.itworkscdn.net/nour4satlive/livestream/playlist.m3u8
+#EXTINF:-1 tvg-id="OTV Lebanon ARB" tvg-name="OTV Lebanon ARB" tvg-language="Arabic" tvg-logo="http://bookback.co/wp-content/uploads/2018/12/OTV-1.jpg" group-title="General",OTV (1080p)
+https://svs.itworkscdn.net/otvlebanonlive/otv.smil/playlist.m3u8
```

Comment:
The link no longer works
<img width="882" alt="Captura de pantalla 2021-01-28 a las 3 45 12" src="https://user-images.githubusercontent.com/7253922/106073533-526fc500-611b-11eb-9635-d7b551e8846a.png">

Reply:
maybe try : https://svs.itworkscdn.net/otvlebanonlive/livestream/playlist.m3u8

Reply:
Thanks for checking but it's still working for me https://streamtest.in/logs/https-svs-itworkscdn-net-otvlebanonlive-otv-smil-playlist-m3u8-90cd8bfb-768a-4b7a-8278-3788feba1819

Maybe it was down before?

Reply:
Or maybe it isn't 24/7

Reply:
Not sure. I think it is 24/7

Reply:
Now it works for me, too.